Dockerfile

```diff
@@ -23,3 +23,5 @@ EXPOSE 50001
 EXPOSE 4224
 
 STOPSIGNAL SIGINT
+
+HEALTHCHECK CMD curl -fSs http://localhost:4224/ || exit 1
```

Comment:
Consider issuing an RPC to 50001 e.g. using:
```
contrib/health_check.py localhost 50001
```

Reply:
I considered it, but the problem is port 50001 isn't opened until all sync is finished. Which took around 16 hours in my case. 16 hours of unhealthy state is confusing I'd say.

Also, port 50001 is actually port 51001 on testnet, while port 4224 will stay the same afaik.

Reply:
> 16 hours of unhealthy state is confusing I'd say.

Agreed.

Reply:
The monitoring port (4224) is changed in Testnet to 14224:
https://github.com/romanz/electrs/blob/5510740a02e4f8140afd2de9a0a5d14c4f1b4eff/src/config.rs#L213

However, we can override it using:
 https://github.com/romanz/electrs/blob/5510740a02e4f8140afd2de9a0a5d14c4f1b4eff/config_spec.toml#L71